pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go

```diff
@@ -267,6 +268,8 @@ func reEnablePITR(pbm *backup.PBM, backup psmdbv1.BackupSpec) (err error) {
 	return
 }
 
+var ErrNoOplogsForPITR = errors.New("There is no oplogs that can cover the date/time or no oplogs at all")
```

Comment:
1. no need to export it.
2. errors should be in lower case
3. no need to do it global

Reply:
changed in e03b30d (but the err is still global due to its usage in different funcs)